### PR TITLE
Prevent divide by 0 runtime, fix supply shuttle

### PR DESCRIPTION
**Pull request by tigercat2000**
This commit adds a small sanity check to the falloff calculations of
lights, to prevent them from dividing by 0. This may cause undiscovered
strangeness, but strangeness is better than filling the runtime log.

This commit also fixes the supply shuttle, which was not spawning any
ordered items. It did this because it had a very basic contents.len check
which the lighting overlays triggered. The sanity check to not spawn stuff
on top of other stuff is now a for(atom) loop on the turfs, which has
snowflake checks for lights and lighting overlays.

TL:DR; Supply shuttle works now, No more spamming runtimes (which may cause things to go insane but who knows, better than dividing by 0)

### CODE REVIEW

**Comment by fj45** [comment]
Divide by zero. 


Dare I ask....what exactly would happen if divide by zero occurred in the code?

**Comment by ZomgPonies** [comment]
It did, causes a runtime error for that proc.
